interactive/printer_test.go

```diff
@@ -48,7 +48,7 @@ func TestPrintMarkedFiles(t *testing.T) {
 	writer.Flush()
 	result := buffer.String()
 	// We don't know the order, as we are using a map to store marked files :/
-	expected := "'d1'\n'd2'\n'd1/d3/f2'\n"
+	expected := "'d1'\n'd1/f1'\n'd2'\n'd1/d3/f2'\n"
```

Comment:
Because `d1` (the parent of `d1/f1`) was already marked. The whole logic you removed above only logs files if their parents are not marked.